### PR TITLE
Update prof_musique.md

### PR DESCRIPTION
**Pull request by NotAnAmbiTurner**
@casey-collab I changed the link to ../french

### CODE REVIEW

**Comment by NotAnAmbiTurner** [comment]
@casey-collab

File: french/telephone/prof_musique.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Comme c'est le weekend, sa mere repond et me demande d'aller lui acheter du pain.
+
+[Returner](/french/feu-de-camp.md)
```
Comment (casey-collab) [inline french/telephone/prof_musique.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by NotAnAmbiTurner** [comment]
@casey-collab : I think I fixed it!

File: french/telephone/prof_musique.md [hunk 2]
```diff
@@ -1 +1,3 @@
 Comme c'est le weekend, sa mere repond et me demande d'aller lui acheter du pain.
+
+[Returner](french/feu-de-camp.md)
```
Comment (casey-collab) [inline french/telephone/prof_musique.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by NotAnAmbiTurner** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**